src/main/resources/stylesheets/snyk_code_suggestion.scss

```diff
@@ -84,6 +80,15 @@ a,
   border-bottom: 3px solid #3474f0;
 }
 
+.example-line {
+  background-color: var(--editor-color);
+}
+
+.example-line > code {
+  font-family: var(--editor-font);
+  font-size: 1.05rem;
+}
```

Comment:
I think this needs to be `monospace`. It's picking up `"JetBrains Mono"`

Reply:
I meant for it to be `JetBrains Mono`, why do you think it should be `monospace`? 
- https://www.jetbrains.com/help/idea/settings-editor-font.html
- https://www.w3schools.com/whatis/tryit.asp?font=JetBrains%20Mono